ugh/src/ugh/fileformats/excel/Excelfile.java

```diff
@@ -2024,7 +2021,7 @@ public boolean CalculateEndPage(DocStruct inStruct)
 								// (physical struct).
 								List<Reference> refs = currentchild
 										.getAllReferences("to");
-								if (refs.size() == 0) {
+								if (refs == null || refs.size() == 0) {
 									// No references set, so set one to the
```

Comment:
Please put conditions in single round brackets:
`if ((refs == null) || (refs.size() == 0)) {`